buildScript/webpack.config.js

```diff
@@ -150,13 +155,18 @@ export default function makeWebpackConfig(config) {
                 // later presets run before earlier for each AST node
                 // use 'es2015', {modules: false}] for es5 with es6 modules
                 presets: [
-                    ['env', {
-                        targets: {
-                            browsers: ['safari >= 9', 'chrome >= 62', 'firefox >= 56', 'edge >= 14']
-                        },
-                        debug: !PROD,
-                        modules: false}], // preserve application module style - in our case es6 modules
-                    'react', 'stage-3'],
+                    ['env',
+                        {
+                            targets: {
+                                browsers: ['safari >= 9', 'chrome >= 62', 'firefox >= 56', 'edge >= 14']
+                            },
+                            debug: !PROD,
+                            modules: false,  // preserve application module style - in our case es6 modules
+                            useBuiltIns : true
```

Comment:
The [docs](https://babeljs.io/docs/plugins/preset-env/#options) says
`This option enables a new plugin that replaces the statement import "babel-polyfill" or require("babel-polyfill") with individual requires for babel-polyfill based on environment.` 

We still have `import 'babel-polyfill'` in Firefly.js